### ISSUE TITLE
What is the url to use in the browser to access the trellis VM?

### ISSUE THREAD

**Issue by tylerfoulkes** [roots/trellis#641]
As far as I'm concerned it seems like everything went fine with the trellis install and I was able to boot the VM using
```
vagrant up
```
However I can't seem to view the site in the browser. I am using "localhost" as the url but I get a 403 forbidden error. I have the local_path in the wordpress_sites.yml file set to "../site".

Any help would be greatly appreciated!

**Comment by swalkinshaw** [comment]
You access your site at whatever host you set for it in `wordpress_sites`. The default is `example.dev`.

**Comment by tylerfoulkes** [comment]
Im getting a "This site can’t be reached" error when I use the site address in my ```wordpress_sites```file.

**Comment by cfxd** [comment]
If you open Virtualbox do you see any VMs running? You should be able to see if there's one running with an address that matches your config file. Make sure that's the one you're visiting and that the VM is actually powered up.

**Comment by tylerfoulkes** [comment]
@cfxd Yes, it says its running fine.

**Comment by iamjoshellis** [comment]
@tylerfoulkes if you've changed the configuration after initialize the vagrant box you'll need to provision again with either:
`vagrant provision` while it's running.
Or `vagrant up --provision` when starting it up.

**Comment by tylerfoulkes** [comment]
@iamjoshellis I tried running the ```vagrant up --provision``` command and it failed at ```TASK [wordpress-install : Install WP]```. I checked out the Fix #639 and I checked my current trellis build and it includes the ```wp_cli_version: 0.24.1``` line in the ```main.yml``` so it seems like everything is up to date. However I am still getting a fail at the following part when I run ```vagrant up --provision```: ```TASK [wordpress-install : Install WP]```

```
System info:
  Ansible 2.1.1.0; Vagrant 1.8.5; Darwin
  Trellis at "Fix #639 - WP 4.6 compatibility: update WP-CLI to 0.24.1"
---------------------------------------------------
PHP Fatal error:  Uncaught phpmailerException: Invalid address:
wordpress@${http_host} in /srv/www/msuexponent.com/current/web/wp/wp-includes
/class-phpmailer.php:946
Stack trace:
#0 /srv/www/msuexponent.com/current/web/wp/wp-includes/pluggable.php(352):
PHPMailer->setFrom('wordpress@${htt...', 'WordPress')
#1 /srv/www/msuexponent.com/current/web/wp/wp-
admin/includes/upgrade.php(392): wp_mail('tylerfoulkes10@...', 'New WordPress
S...', 'Your new WordPr...')
#2 /srv/www/msuexponent.com/current/web/wp/wp-admin/includes/upgrade.php(95):
wp_new_blog_notification('msuexponent.com', 'http://${HTTP_H...', 1, 'The
password yo...')
#3 phar:///usr/local/bin/wp/php/commands/core.php(763):
wp_install('msuexponent.com', 'admin', 'tylerfoulkes10@...', true, '',
'admin')
#4 phar:///usr/local/bin/wp/php/commands/core.php(573):
Core_Command->_install(Array)
#5 [internal function]: Core_Command->install(Array, Array)
#6 phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php(67):
call_user_func(Array, Array, Array)
#7 [internal functio in /srv/www/msuexponent.com/current/web/wp/wp-includes
/class-phpmailer.php on line 946
failed: [default] (item=msuexponent.com) => {"changed": true, "cmd": ["wp", "core", "install", "--allow-root", "--url=http://${HTTP_HOST}", "--title=msuexponent.com", "--admin_user=admin", "--admin_password=admin", "--admin_email=tylerfoulkes10@gmail.com"], "delta": "0:00:06.093297", "end": "2016-08-30 23:45:25.713351", "failed": true, "item": "msuexponent.com", "rc": 255, "start": "2016-08-30 23:45:19.620054", "stderr": "PHP Fatal error:  Uncaught phpmailerException: Invalid address: wordpress@${http_host} in /srv/www/msuexponent.com/current/web/wp/wp-includes/class-phpmailer.php:946\nStack trace:\n#0 /srv/www/msuexponent.com/current/web/wp/wp-includes/pluggable.php(352): PHPMailer->setFrom('wordpress@${htt...', 'WordPress')\n#1 /srv/www/msuexponent.com/current/web/wp/wp-admin/includes/upgrade.php(392): wp_mail('tylerfoulkes10@...', 'New WordPress S...', 'Your new WordPr...')\n#2 /srv/www/msuexponent.com/current/web/wp/wp-admin/includes/upgrade.php(95): wp_new_blog_notification('msuexponent.com', 'http://${HTTP_H...', 1, 'The password yo...')\n#3 phar:///usr/local/bin/wp/php/commands/core.php(763): wp_install('msuexponent.com', 'admin', 'tylerfoulkes10@...', true, '', 'admin')\n#4 phar:///usr/local/bin/wp/php/commands/core.php(573): Core_Command->_install(Array)\n#5 [internal function]: Core_Command->install(Array, Array)\n#6 phar:///usr/local/bin/wp/php/WP_CLI/Dispatcher/CommandFactory.php(67): call_user_func(Array, Array, Array)\n#7 [internal functio in /srv/www/msuexponent.com/current/web/wp/wp-includes/class-phpmailer.php on line 946", "stdout": "", "stdout_lines": [], "warnings": []}
```

**Comment by swalkinshaw** [comment]
There's already issues open for this and it's not related to your original question.
